uni-stark/tests/mul_air.rs

```diff
@@ -26,6 +26,8 @@ impl<AB: AirBuilder> Air<AB> for MulAir {
 
 #[test]
 #[ignore] // TODO: Not ready yet.
+#[allow(clippy::items_after_statements)]
+#[allow(clippy::upper_case_acronyms)]
```

Comment:
Hm maybe we should change the names, I didn't realize there was a clear convention for acronyms. I could do the renames later since it's an easy change but could lead to conflicts.

Reply:
Well, this is from `clippy::pedantic`, so take the suggestions with a grain of salt.

Specifically, I ran with
```bash
#!/bin/bash
cargo +nightly clippy --all-targets -- \
    -D clippy::pedantic \
    -A clippy::inline-always \
    -A clippy::similar-names \
    -A clippy::module-name-repetitions \
    -A clippy::missing-errors-doc \
    -A clippy::missing-panics-doc
```

I'd say, if you are one for conventions, `clippy::module-name-repetitions` is the one to follow first.

Reply:
Hmm, since we have small crates, we're mostly re-exporting so that users see e.g. `p3_field::AbstractField`. I guess that makes the clippy a sort of false positive, and we probably wouldn't want users to write `p3_field::AbstractField`; either `field::Abstract` (which isn't supported unless rebinding) or `AbstractField` seems better.

Also it seems like `module-name-repetitions` might be more of a weak convention? It doesn't seem to be explicit in https://rust-lang.github.io/api-guidelines/naming.html (although I guess the iterator section hints at it). Real usage seems mixed, like `rand::distributions` structs don't contain `Distribution`, but `rand::rngs` structs do contain `Rng`.

Reply:
Yes, I wouldn't worry too much about the repetition.

You have good reasons here to keep it.